cmake/arbor-config.cmake.in

```diff
@@ -44,6 +44,14 @@ function(_append_property target property)
     endif()
 endfunction()
 
+set(ARB_VECTORIZE @ARB_VECTORIZE@)
+set(ARB_ARCH @ARB_ARCH@)
+set(ARB_MODCC_FLAGS @ARB_MODCC_FLAGS@)
+set(ARB_CXX @CMAKE_CXX_COMPILER@)
+set(ARB_CXX_FLAGS @CMAKE_CXX_FLAGS@)
+set(ARB_CXXOPT_ARCH @ARB_CXXOPT_ARCH@)
```

Comment:
I know why these are here, but it feels a bit ugly: they act as sort of backdoor parameters for the standalone catalogue building. I worry that it's a bit fragile, but it will all be revisited in mechanism-ABI land, so I'll try not to.

Reply:
Exactly, this is the plan.